### DISCUSSION TITLE
Why sandbox process not work with ts-node-dev ?

### DISCUSSION THREAD

**Discussion by yakupbeyoglu** [taskforcesh/bullmq#1454] [Q&A] [answered]
In the package.json, when we have ts-node as the start script for npm start, works without any error. However when we change it with ts-node-dev for watching changes, the sandbox process opens but not works, also sandbox process is not closed which means loads to CPU completely.

**Comment by manast** [comment, marked as answer]
All I have to say regarding this issue is written here: https://blog.taskforce.sh/using-typescript-with-bullmq/

**Reply by yakupbeyoglu** [reply]
Thanks for your reply. Its works with your solutions.

Also, you can find another solution with **nodemon** below.

```
  "scripts": {
    "start": "ts-node --respawn --transpile-only src/index.ts",
    "dev": "nodemon src/index.ts"
  }
```
In this way, .js file is not generated on the src directory.
To run watch you should call `npm run dev`
